docs/integrations.rst

```diff
@@ -317,6 +317,10 @@ Zed Attack Proxy
 ----------------
 ZAP XML report format.
 
+ESLint
+----------------

```

Comment:
Please limit the hyphens to the same number of character that's are in the 'ESLint'

Reply:
Done